src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -88,27 +131,22 @@ protected static ConfigDef baseConfigDef() {
       int order = 0;
       configDef
           .define(TYPE_NAME_CONFIG, Type.STRING, Importance.HIGH,
-                  "The Elasticsearch type name to use when indexing.",
+                  TYPE_NAME_DOC,
                   group, ++order, Width.SHORT, "Type Name")
```

Comment:
I think checkstyle just cares about line length. Must admit if i'm going to split args over multiple lines then i'd prefer 1 arg per line as it makes it much easier for me to grok. Subjective, though